pkg/driver/node.go

```diff
@@ -82,6 +86,10 @@ func (d *Driver) NodePublishVolume(ctx context.Context, req *csi.NodePublishVolu
 				return nil, status.Errorf(codes.InvalidArgument, "Volume context property %q must be an absolute path", k)
 			}
 			subpath = filepath.Join(subpath, v)
+		case "storage.kubernetes.io/csiprovisioneridentity":
+			continue
```

Comment:
Curious why this would be set in volume context.

Reply:
I'm not sure how to get rid of this. Kubernetes sets this for all dynamically provisioned volumes